api/Window.json

```diff
@@ -4709,7 +4709,7 @@
               "version_added": "7.0"
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "57"
```

Comment:
This is the `onappinstalled` event handler attribute. Is web app installation via WebView really supported? Seems like it likely isn't, although the event handler attribute might still be incorrectly exposed I guess. Can you look a bit closer and see if this needs a note?

Reply:
The collector gets data for WebView specifically, so this __is__ actually supported in WebView, though it's probably not supposed to be.

Reply:
Okay, probably a case of a missing feature flag for the event handler attribute, although the event will never be fired.